frontend/src/App.jsx

```diff
@@ -1,22 +1,58 @@
-import './App.css'
 import HomePage from "./components/homePage/HomePage.jsx"
-
+import ForYouPage from "./components/forYouPage/ForYouPage.jsx"
+import NearbyPage from "./components/nearbyPage/NearbyPage.jsx"
+import EventAttendingPage from "./components/eventAttendingPage/EventAttendingPage.jsx"
+import FavoriteEventsPage from "./components/favoriteEventsPage/FavoriteEventsPage.jsx"
+import InitialPage from "./components/initialPage/InitialPage.jsx"
+import SignupPage from "./components/signUpPage/SignUpPage.jsx"
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
+{/* testing for user auth */}
+import LoginForm from "./components/LoginForm/LoginForm.jsx"
+import SignupForm from "./components/SignupForm/SignupForm.jsx"
+import { useState, useEffect } from 'react';
+import { UserContext } from './UserContext';
+
+
 function App() {
 
+  const [user, setUser] = useState(() => {
+    // Retrieve the user data from storage or set it to null if not found
+    const storedUser = localStorage.getItem('user');
+    // return storedUser ? JSON.parse(storedUser) : null;
+    return null
```

Comment:
This is where user is initialized in your state. The logic should be search in cookie, if it does exist in cookie, retrieve from cookie. Why is it returning null here?

Reply:
It was for debugging purposes